atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -278,10 +278,13 @@ def get_source_registry(workflow, fallback=NO_FALLBACK):
             return fallback
         raise
 
-    return {
+    regdict = {
         'uri': RegistryURI(source_registry['url']),
         'insecure': source_registry.get('insecure', False)
     }
+    if source_registry.get('auth'):
+        regdict['secret'] = source_registry['auth']['cfg_path']
```

Comment:
set 'secret' to None in case auth isn't defined

Reply:
Fixed.